_about/notes/0.5.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Alpha2 Traffic Rules**. Add support for an updated model of routing and destination rules. TODO: Doc link?
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use new Kubernetes 1.9 mutating webhook functionality to provide automatic
+pod-level proxy injection. TODO: Doc link?
+
+## Mixer adapters
+
+- **Open Policy Agent**. Mixer now has an authorization adapter implementing the [open policy agent](http://www.openpolicyagent.org) model,
+providing a flexible fine-grained access control mechanism. TODO: Doc link?
+
+- **RBAC**. Mixer now has a resource-based access control adapter. TBD: Doc link?
+
+- **Fluentd**. Mixer now has an adapter for [fluentd](https://www.fluentd.org) telemetry collection. TODO: Doc link?
+
+- **Stdio**. The stdio adapter now lets you log to files with support for log rotation & backup, along with a host
+of controls.
+
+## Security
+
+- **Bring Your Own CA**. There have been many enhancements to the 'bring your own CA' feature. TODO: Doc link?
+
+- **PKCS8**. Add support for PKCS8 keys to Istio PKI.
+
+## Other
+
+- **Component Logging**. Istio components now offer a rich set of command-line options to control local logging, including
+common support for log rotation. 
```

Comment:
can we put a link to teach people how to view the logs?

Reply:
We don't have docs for this yet. I'm working at the moment on extracting a fresh set of CLI reference docs to put on istio.io. Once we have those, we can point people to those docs.

Reply:
ok SGTM